tests/python/frontend/darknet/test_forward.py

```diff
@@ -0,0 +1,437 @@
+"""
+Compile Darknet Models
+=====================
+This article is a test script to test darknet models with NNVM.
+All the required models and libraries will be downloaded from the internet
+by the script.
+"""
+import os
+import requests
+import sys
+import urllib
+import numpy as np
+import tvm
+from tvm.contrib import graph_runtime
+from nnvm import frontend
+from nnvm.testing.darknet import LAYERTYPE
+from nnvm.testing.darknet import __darknetffi__
```

Comment:
I think nnvm.testing.darknet should be moved to tvm.relay.testing.darknet like the tensorflow frontend.